chef/data_bags/crowbar/template-provisioner.json

```diff
@@ -9,6 +9,8 @@
       "access_keys": "",
       "shell_prompt": "USER@HOST:CWD SUFFIX",
       "enable_pxe": true,
+      "self_update": false,
+      "self_update_url": "http://updates.suse.com/sle12/$arch",
```

Comment:
Shouldn't we default this to a repo that is mirrored on the admin server, like all the other repos we have there?